controller_manager/scripts/spawner

```diff
@@ -85,21 +82,20 @@ def parse_args(args=None):
     parser.add_argument('--timeout', metavar='T', type=float, default=30,
                         help='how long to wait for controller_manager services when starting up [s] (default: 30)')
     parser.add_argument('--shutdown-timeout', metavar='T', type=float, default=30,
-                        help='how long to wait for controller_manager services when shutting down [s] (default: 30)')
+                        help='DEPRECATED: this argument has no effect anymore.')
     parser.add_argument('controllers', metavar='controller', nargs='+',
                         help='controllers to load')
     return parser.parse_args(args=args)
 
 def main():
-    global unload_controller_service,load_controller_service,switch_controller_service,shutdown_timeout
+    global unload_controller_service,load_controller_service,switch_controller_service
 
     args = parse_args(rospy.myargv()[1:])
 
     wait_for_topic = args.wait_for
     autostart = 1 if not args.stopped else 0
     robot_namespace = args.namespace or ""
     timeout = args.timeout
-    shutdown_timeout = args.shutdown_timeout
 
     rospy.init_node('spawner', anonymous=True)
 
```

Comment:
And print a deprecation warning if the value was set (i.e. is not None).

Reply:
Done